functions.sh

```diff
@@ -306,3 +306,21 @@ aix_ld_hack() {
             ;;
     esac
 }
+
+#
+# Safety DB IDs to be ignored when using pyOpenSSL older than 17.5.0.
+#
+add_ignored_safety_ids_for_pyopenssl_pre17_5_0() {
```

Comment:
I don't know what to say....

I would not ignore this and just upgrade pyopenssl.

These CVEs don't affect 0.13.1